shap/plots/colors/_colorconv.py

```diff
@@ -1,475 +1,103 @@
+### ------------ vendored from skimage.color.colorconv -------------
 # This is a small chunk of code from the skimage package. It is reproduced
-# here because all we need is a couple color conversion routines, and adding
-# all of skimage as dependency is really heavy.
+# here because all we need is a couple color conversion routines (lab2rgb, lch2lab)
+# and adding all of skimage as dependency is really heavy.
 
-# Disable linting on vendored code
-# ruff: noqa
 
 # Copyright (C) 2019, the scikit-image team
```

Comment:
Shall we follow their [LICENSE.txt](https://github.com/scikit-image/scikit-image/blob/main/LICENSE.txt) file?

```suggestion
Copyright: 2009-2022 the scikit-image team
License: BSD-3-Clause
```

Reply:
Good spot, done. I committed separately to ensure the `#` is retained